### DISCUSSION TITLE
❗ Add a drawer component to orbit

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1121] [New components Ideas]
## Description

In many of our applications, we have a modal that is docked to the sides. This could be a orbit component. 

![image](https://user-images.githubusercontent.com/38871812/194601524-b08488ac-5ce8-4260-b10e-deb47d8c030e.png)
![image](https://user-images.githubusercontent.com/38871812/194601576-18081f37-7d44-4b8e-9e41-b29b282c12c1.png)

**Comment by patricklafrance** [comment]
I suggest moving this to a Github discussion.
